plugins/scanners/fossid/src/main/kotlin/FossId.kt

```diff
@@ -275,9 +275,8 @@ class FossId internal constructor(
                         "path '${pkg.vcsProcessed.path}' set and scanning cannot be limited to paths.",
                     severity = Severity.WARNING
                 )
-                val provenance = RepositoryProvenance(pkg.vcsProcessed, pkg.vcsProcessed.revision)
 
-                return createSingleIssueResult(issue, provenance)
+                return createSingleIssueResult(issue, UnknownProvenance)
```

Comment:
The provenance parameter could now be removed from the `createSingleIssueResult` function as it is always the same.

But maybe it would be better to do the opposite change and provide the rejected provenance also in the two cases above to have more information in the ORT result?

Reply:
> provide the rejected provenance also in the two cases above to have more information in the ORT result?

I was considering the same, but was uncertain about the intended semantics of `ScanResult.provenance` which is documented as "Provenance information about the *scanned* source code" (emphasis mine). As nothing is actually scanned in these cases, it felt wrong to return a known provenance. Do you recall the intended semantics here @mnonnenmacher?

Reply:
> I agree with Martin, it feels we are losing information here by putting `UnknownProvenance`.

Well, we already "lost" information before this PR as some code paths already returned `UnknownProvenance`, and I just wanted to align that. But after having a chat with @mnonnenmacher I'm now doing some further refactoring to align this in the opposite way.